app/controllers/posts_controller.rb

```diff
@@ -704,10 +704,10 @@ def create_params
 
     end
 
-    result = params.permit(*permitted).tap do |whitelisted|
-      whitelisted[:image_sizes] = params[:image_sizes]
+    result = params.permit(*permitted).tap do |allowlisted|
```

Comment:
Nitpick: I prefer `allowed`.

```suggestion
    result = params.permit(*permitted).tap do |allowed|
```